pkg/api/testapi/testapi.go

```diff
@@ -63,10 +63,9 @@ var (
 )
 
 type TestGroup struct {
-	// the first element in the group is meant to be the preferred version
-	externalGroupVersions []unversioned.GroupVersion
```

Comment:
It seems you were undoing your changes, the testapi used to only have one externalGroupVersion, am I right?